src/Server.ts

```diff
@@ -52,14 +51,16 @@ async function main() {
 		app.use(Sentry.Handlers.requestHandler());
 		app.use(Sentry.Handlers.tracingHandler());
 	}
-	await Promise.all([api.start(), cdn.start(), gateway.start()]);
 	if (Config.get().sentry.enabled) {
 		app.use(Sentry.Handlers.errorHandler());
 		app.use(function onError(err: any, req: any, res: any, next: any) {
 			res.statusCode = 500;
 			res.end(res.sentry + "\n");
 		});
 	}
+
+	server.listen(port);
```

Comment:
These need to be between the Sentry blocks, as controllers need to be registered before adding the error handler.
Source: https://docs.sentry.io/platforms/node/guides/express/